assignments/prototypes.js

```diff
@@ -14,14 +14,28 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
-
+function GameObject(property) {
+  this.createdAt = property.createdAt;
+  this.dimensions = property.dimensions;
+  this.destroy = function() {
+    return `${this.name} was removed from the game`
+  };
+};
 /*
   === CharacterStats ===
   * healthPoints
   * name
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(attributes) {
+  GameObject.call(this, attributes);
+  this.healthPoints = attributes.healthPoints;
+  this.name = attributes.name;
+  this.takeDamage = function() {
+    return `${this.name} took damage.`;
+  };
+};
 
```

Comment:
again here you should be using a prototype function outside the CharacterStats function here is the example for this one.
```suggestion
// Sets up inheritance with GameObject
CharacterStats.prototype = Object.create(GameObject.prototype);

CharacterStats.prototype.takeDamage = function () {
  return `${this.name} took damage.`;
};
```